activelearning/dataset/grid.py

```diff
@@ -57,7 +57,7 @@ def preprocess(self, X, y):
         - normalizes the scores
         - normalizes the states
         """
-        states = X / self.grid_size
+        states = X / self.grid_size * 2 - 1  # normalize to [-1; 1]
```

Comment:
Same as above

Reply:
what exactly do you think is the problem with this?
do you think that the normalization before was ok?

Reply:
See my comments on the other thread. I think it is very important here to write a good docstring.